site2/docs/functions-develop-api.md

```diff
@@ -102,6 +102,39 @@ public class ExclamationFunction implements Function<String, String> {
 
 For more details, see [code example](https://github.com/apache/pulsar/blob/master/pulsar-functions/java-examples/src/main/java/org/apache/pulsar/functions/api/examples/ExclamationFunction.java).
 
+The return type of the `Function` can be wrapped in a Pulsar `Record` generic which gives more control on the output message (topic, schema, properties, ...)
```

Comment:
```suggestion
The return type of the function can be wrapped in a generic `Record` which gives you more control over the output message, such as topic, schema, properties, and so on.
```

Reply:
I generally agree with the proposed changes except for inverting `generic` and `Record`. Generic here is for a Java generic (noun), it's not an adjective.

Reply:
Got your point. I've updated my suggestion.